tests/smoketest/smoketest.sh

```diff
@@ -82,8 +93,7 @@ oc logs "$(oc get pod -l "smart-gateway=default-cloud1-ceil-event" -o jsonpath='
 echo
 
 echo "*** [INFO] Logs from smart gateway operator..."
-oc logs "$(oc get pod -l app=smart-gateway-operator -o jsonpath='{.items[0].metadata.name}')" -c ansible
-echo
+oc logs "$(oc get pod -l app=smart-gateway-operator -o jsonpath='{.items[0].metadata.name}')" -c ansible echo
```

Comment:
Do you mean for a `;` to separate `ansible` and `echo` here?

Reply:
nope!

Reply:
This doesn't seem right. I'd expect it to return: `error: only one of -c or an inline [CONTAINER] arg is allowed`

Reply:
`-c ansible` is specifyng the container from which to print logs. Also I did mean to put a semicolon there